libseccomp/Cargo.toml

```diff
@@ -19,3 +20,6 @@ pkg-config = "0.3.19"
 
 [package.metadata.docs.rs]
 rustc-args = ["--cfg", "docsrs"]
+
+[features]
+scmpsyscall_new = ["cfg-if"]
```

Comment:
I prefer `const-syscall`, but actually I'm not sure the best name.
What do you think?

Reply:
Yes, naming features after use case rather then function name is more future proof.

Reply:
True. If you like `const-syscall`, please fix it. Of course, if you have a better name, please let me know:)